lib/elixir/lib/calendar.ex

```diff
@@ -67,7 +67,7 @@ defmodule Calendar do
   @doc """
   Converts the given structure into a string according to the calendar.
   """
-  @callback to_string(Date.t | NaiveDateTime.t | DateTime.t) :: String.t
+  @callback to_string(structure) :: String.t when structure: Date.t | DateTime.t | NaiveDateTime.t
```

Comment:
Sorry i wasnt clear. The whole point of the pr was to avoid arg0. I can submit a new pr with whatyouhide version